DTR.md

```diff
@@ -89,32 +69,10 @@ Completed DTR and start coding (on day 2 - 2021 Jul 27, Tue).
 Day 1: Wireframes; Frontend stories (complete)
 Day 2: Backend stories and some coding.
 
-### Re-DTR 2021 Jul 30, Fri
+### Re-DTR
 
 **Things that are going well**
-- Moving along... knocked on major challenges (React up and running, recommendation system functioning, OAuth Google)
-- Happy 21st Richard, don't forget to Drink This 🍸 🍹
-- Not as stressful in this project
-- Divide and conquer working well
-- Everyone is contributing and being confident
+- 
 
 **Things to improve**
-- Better on PRs (Jermaine)
-- Have constant communication (Zach)
-- Challenge: Figuring out how the hand-off goes; Making the FE and BE work smoothly, going into production (Molly)
-- Keep kanban project board updated (Richard)
-
-**Day 5 - Goals**
-Everyone: Work on your Terminal profile and resume
-
-Richard: Birthday weekend (not a great start with a fly flying into mouth; Google button will be done today
-
-Molly: Not available on Sat, but back fully online on Sun; Get good deployment to heroku for production
-
-Taija: Weekend is free, just onboarding babysitter
-
-Zach: Online 23/7
-
-Jermaine: Sat busy, Sun free
-
-Mark: Weekend is free (just have to take a friend to airport Sun early morning)
+- 
```

Comment:
We need to make sure that this stuff doesn't get deleted. Per conversation, sounds like we need to merge the landing page from @twarbelow and @redferret prior to this so that the base branch for this is up to date. Good thing to check in on this afternoon?

Reply:
Latest DTR pushed looks good.